fix(cgpt): get an aligned variable

vbatts opened the pull request:
Since the GptEntry is packed, the pointer is possibly unaligned.

This has been a warning, but on gcc 9.3 it errors.

Signed-off-by: Vincent Batts <vbatts@kinvolk.io>

# fixes the compiler error due to aligning to a packed struct


# How to use

[ describe what reviewers need to do in order to validate this PR ]


# Testing done

compiling and running the `rootdev` command

dongsupark reviewed (commented):
Code looks good. Thanks!

However, it is the same issue.
The base branch should be `flatcar-master` instead of `master`.
Sorry again!

Commit:
fix(cgpt): get an aligned variable

Since the GptEntry is packed, the pointer is possibly unaligned.

This has been a warning, but on gcc 9.3 it errors.

Signed-off-by: Vincent Batts <vbatts@kinvolk.io>

vbatts commented:
rebase and pointed to flatcat-master
